docs/admin.md

```diff
@@ -3,3 +3,13 @@
 The Flarum Admin Dashboard is a user-friendly interface for managing your forum.
 It is only available to users in the "Admin" group.
 To access the Admin dashboard, Click on your **Name** at the at the top right of the screen, and select **Administration**.
+
+The Admin Dashboard has the following sections, being:
+- **Dashboard** - Shows the main Admin Dashboard, containing statistics and other relevant information.
+- **Basics** - Shows the options to set basic forum details such as Name, Description, and Welcome Banner.
+- **Email** - Allows you to configure your E-Mail settings. Refer [here](https://docs.flarum.org/mail) for more information.
+- **Permissions** - Shows the permissions for each user group, and allows you to configure global and specific scopes.
+- **Appearance** - Allows you to customize the forum's colors, branding and add additional CSS for customization.
+- **Users** - Provides you with a paginated list of all the users in the forum, and grants you the ability to edit the user or take administrative actions.
+
+Apart from the above mentioned sections, the Admin Dashboard also allows you to manage your Extensions (including the flarum core extensions such as Tags) under the _Features_ section. Extensions which modify the forum theme, or allow you to use multiple languages are categorized under the _Themes_ and _Languages_ section respectively.
```

Comment:
Since we always refer to them as categories, let's do that here as well.
```suggestion
Apart from the above mentioned sections, the Admin Dashboard also allows you to manage your Extensions (including the Flarum core extensions such as Tags) under the _Features_ category. Extensions which modify the forum theme, or allow you to use multiple languages are categorized under the _Themes_ and _Languages_ categories respectively.
```